content/tracing/languages/java.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Tracing Java Applications
+kind: Documentation
+further_reading:
+- link: "https://github.com/DataDog/dd-trace-java"
+  tag: "Github"
+  text: Source code
+- link: "/tracing/"
```

Comment:
Should this link be `/tracing/java`?

Reply:
Nop, this link is a the bottom of the page to point to main `/tracing` doc